File: src/test/fixtures/symbols.ts
const myConst = 1;

function myFunction() {}

function myOuterFunction() {
  function myInnerFunction() {
    return 1;
  }
}

class MyClass {
  myMethod() {
    return 1;
  }
}

interface MyInterface {
  x: number;
}

enum MyEnum {
  A,
  B,
}

File: src/test/fixtures/rust/symbols.rs
const MY_CONST: i32 = 1;

fn my_function() {}

fn my_outer_function() {
    fn my_inner_function() {
        let _ = 1;
    }
}

struct MyStruct {
    x: i32,
}

impl MyStruct {
    fn my_method(&self) -> i32 {
        self.x
    }
}

trait MyTrait {
    fn my_trait_method(&self) -> i32;
}

enum MyEnum {
    A,
    B,
}
